lib/elixir/src/elixir_fn.erl

```diff
@@ -99,6 +99,12 @@ capture_require(Meta, {{'.', _, [Left, Right]}, RequireMeta, Args} = Expr, E, Se
 handle_capture({local, Fun, Arity}, _Meta, _Expr, _E, _Sequential) ->
   {local, Fun, Arity};
 handle_capture({remote, Receiver, Fun, Arity}, Meta, _Expr, E, _Sequential) ->
+  if
+    is_atom(Receiver) ->
+      elixir_lexical:record_remote(Receiver, Fun, Arity, ?m(E, function), ?line(Meta), ?m(E, lexical_tracker));
```

Comment:
That's an integration test through another feature. I think we need one in the lexical tracker. :) I don't want to run the mix suite to find out bugs in the Elixir code. :)